pages/team.js

```diff
@@ -66,6 +115,27 @@ export default class Team extends Component {
     const moderators = map(prop('osm_id'), team.moderators)
     const isUserModerator = contains(parseInt(this.props.user.uid), moderators)
 
+    let members = team
```

Comment:
I think you mean `team.members`?
```suggestion
    let members = team.members
```